examples/examples.py

```diff
@@ -60,7 +60,7 @@
     - timescale: with database and grafana (note: you need docker installed)
     """
     data_format = "local_db"  # "local_db" or "timescale"
-    example = "small"
+    example = "small_learning_1"
```

Comment:
I would't have the learning example as default on the main, since torch is not installed by default